webpack/scenes/Subscriptions/Manifest/Manifest.scss

```diff
@@ -1,3 +1,8 @@
 .manifest-actions > * {
   margin: 10px 5px 0 0 ;
-}
\ No newline at end of file
+}
+
+.modal-body {
```

Comment:
Do we always want the scrollbar?  If so should we consider adding this to `overrides.scss`?

Reply:
I have no otherwise situations right now, although I think that in general modal shouldn't have with too much data that requires scrolling. 

@Rohoover Any thoughts?

Reply:
That's fair.  I think if we ever need this anywhere else we can just move it to overrides.scss.